### ISSUE TITLE
Tandem Fade Overline / Subhead Light Mode Color Contrast

### ISSUE THREAD

**Issue by tommyneu** [unlcms/project-herbie#955]
_Accidentally made this issue in the wrong repo_

Tandem fade's overline and subhead have insufficient color contrast in light mode

Text Color: .unl-dark-gray
Background Color: .--bg-lighter-gray
